code/controllers/subsystem/persistent_paintings.dm

```diff
@@ -128,7 +133,7 @@ SUBSYSTEM_DEF(persistent_paintings)
 		"diamond" = PATRONAGE_AMAZING_FRAME,
 		"rainbow" = PATRONAGE_SUPERB_FRAME,
 		"supermatter" = PATRONAGE_LEGENDARY_FRAME
```

Comment:
```suggestion
		"supermatter" = PATRONAGE_LEGENDARY_FRAME,
```

if you're fixing indentations may as well make the full list perfect with the trailing comma